include/arch/arm/arch/object/vcpu.h

```diff
@@ -204,9 +206,12 @@ static inline VPPIEventIRQ_t irqVPPIEventIndex(irq_t irq)
 
 #else /* end of CONFIG_ARM_HYPERVISOR_SUPPORT */
 
-/* used in boot.c with a guard, use a marco to avoid exposing vcpu_t */
+/* used in boot.c with a guard, use a macro to avoid exposing vcpu_t */
```

Comment:
I'd remove the whole comment to be honest, because it is misleading. The compile error for exposing vcpu_t is easily fixed by moving `typedef struct vcpu vcpu_t;` to the header file. But then you'll run into the real issue of `vcpu_boot_init()` not existing if HYP is disabled.

Reply:
I'm happy to remove the comment, I have to admit that I wasn't even thinking about what it said, I just saw the typo and wanted to fix it.